getsolus/packages
# obs-studio: Update to 30.0.2

**Summary**

Resolves #1001

Release notes can be found [here](https://github.com/obsproject/obs-studio/releases/tag/30.0.2).

**Test Plan**

 1. Open OBS and test video, audio, and mic input.
 2. record to file and confirm it's working
 3. Add browser source and confirm that's working

**Checklist**
